library/src/main/java/com/datatorrent/lib/io/fs/AbstractFileInputOperator.java

```diff
@@ -25,6 +25,7 @@
 import java.io.InputStream;
 import java.io.InputStreamReader;
 import java.io.Serializable;
+
```

Comment:
you should avoid adding or removing blank lines, or in general avoid any formatting changes around the code you are touching.